test/powershell/Modules/Microsoft.PowerShell.Management/Get-ChildItem.Tests.ps1

```diff
@@ -226,14 +226,31 @@ Describe "Get-ChildItem" -Tags "CI" {
         }
 
         It 'Works with Windows volume paths' -Skip:(!$IsWindows) {
-            $volume = (Get-Volume -DriveLetter $env:SystemDrive[0]).Path
-            $items = Get-ChildItem -LiteralPath "${volume}Windows"
-            Write-Verbose -Verbose "Trying files in '${volume}Windows'"
-            if (-not $items) {
+            $winPath = $env:windir
+            if (! $winPath) {
+                Write-Verbose -Verbose "variable windir is null"
```

Comment:
Do we need this verbose, it will be outputted by the Set-ItResult

Reply:
I think of verbatim